react/features/large-video/components/LargeVideo.web.js

```diff
@@ -37,6 +37,13 @@ export default class LargeVideo extends Component {
                         src = '' />
                 </div>
                 <span id = 'remoteConnectionMessage' />
+                <div className = 'video_blurred_container'>
```

Comment:
Probably I'm missing some context here but do we really need a copy of the large video? Can't we just apply some css to the large video?